gradle/libs.versions.toml

```diff
@@ -1,6 +1,7 @@
 [versions]
 opentelemetry = "1.31.0"
-opentelemetry-alpha = "1.29.0-alpha"
+opentelemetry-alpha = "1.31.0-alpha"
+opentelemetry-semconv-version = "1.21.0-alpha"
```

Comment:
Nit: I don't think we need to add the word `version` on these var names.

Reply:
I only added it to distinguish it from the same-named thing `opentelemetry-semconv` in the `[libraries]` section, but now I'm realizing that it doesn't matter and `version` is redundant. I'll see about removing it.